assets/thanos-ruler/cluster-role.yaml

```diff
@@ -23,3 +23,11 @@ rules:
   - securitycontextconstraints
   verbs:
   - use
+- apiGroups:
```

Comment:
same thing for thanos-ruler service account.

Reply:
same comment as above